clients/python/basic.receiver.py

```diff
@@ -0,0 +1,63 @@
+from basic import BasicCommon
```

Comment:
why a "dot" in the file name?

Reply:
The "Config" class determines the running client by getting the python file name and splitting using "." (dot). The first part of the returned array is the "client name".

It is used to help automatically reading the "basic" (this particular client name) section of the config.ini file.

That was my initial/quick idea to make it simpler to automatically identify the section to use from the config.ini file. I am open for changing it in the future.

Reply:
ok! the dot for python is tricky, i think you can not import this file because it fails, only  works as executable, but ok for now.